### PR TITLE
Enable existing SignalR tests

### PR DESCRIPTION
**Pull request by moozzyk**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
:shipit: assuming Travis and Appveyor pass

**Comment by BrennanConroy** [comment]
Test causing hang is https://github.com/aspnet/SignalR-Server/blob/623bd9fdf339da2990e6c75f3dbc6a9e2fa2499f/test/Microsoft.AspNetCore.SignalR.Messaging.Tests/ScaleoutStreamFacts.cs#L241 changing method to `async void` then adding some awaits will cause it to fail instead of hang :)

**Comment by moozzyk** [comment]
Interesting. I also noticed that the tests pass when running with `-parallel none`. I will continue to investigate tomorrow.

**Comment by moozzyk** [comment]
:up::date:

File: test/Microsoft.AspNetCore.SignalR.Messaging.Tests/ScaleoutStreamFacts.cs [hunk 1]
```diff
@@ -248,14 +248,14 @@ public void OpenQueueErrorOpenQueue()
...
+                await Task.Delay(50).ConfigureAwait(false);
                 x++;
             },
             null);
 
             Task t1 = stream.Send(async _ =>
             {
-                await Task.Delay(50);
+                await Task.Delay(50).ConfigureAwait(false);
                 await TaskAsyncHelper.FromError(new Exception());
```
Comment (BrennanConroy) [inline test/Microsoft.AspNetCore.SignalR.Messaging.Tests/ScaleoutStreamFacts.cs]: This one doesn't need ConfigureAwait?

**Comment by BrennanConroy** [comment]
:shipit:

File: test/Microsoft.AspNetCore.SignalR.Messaging.Tests/ScaleoutStreamFacts.cs (hunk 1 above)
Comment (moozzyk) [inline test/Microsoft.AspNetCore.SignalR.Messaging.Tests/ScaleoutStreamFacts.cs]: No - we create a task that is already completed.
